### PR TITLE
Updating the UniversalViewer to support the new posterImageExpanded config. value

### LINKED ISSUE #3075: Recording manifests should display an attached image FileSet if there is one as the background of the viewer
This means a Recording will have member_ids which are four FileSets: XML File/Image/Two sound Files. The manifest builder will say "I'm a recording, I have an image FileSet, make it my background. Don't do anything with the XML file."

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #3075 fully by building the Universal Viewer using my branch on https://github.com/UniversalViewer/universalviewer/pull/674

### CODE REVIEW

File: public/uv/lib/AVComponent.js [hunk 1]
```diff
@@ -124,14 +124,16 @@ var IIIFComponents;
             //private _lastCanvasWidth: number | undefined;
             _this._lowPriorityFrequency = 250;
             _this._mediaSyncMarginSecs = 1;
-            _this._rangeSpanPadding = 0.0;
+            _this._rangeSpanPadding = 0.25;
```
Comment (tpendragon) [inline public/uv/lib/AVComponent.js]: This removes all our local patches to the AVComponent, which means it'll start skipping content again.

File: public/uv/lib/uv-av-extension-dependencies.js [hunk 2]
```diff
@@ -10,7 +19,25 @@ define(function () {
         return isFormatAvailable(formats, 'application/dash+xml');
     }
     function canPlayHls() {
-      return true;
```
Comment (tpendragon) [inline public/uv/lib/uv-av-extension-dependencies.js]: This makes `hls.js` not load anymore.

File: public/uv/lib/AVComponent.js (hunk 1 above)
Comment (jrgriffiniii) [reply public/uv/lib/AVComponent.js]: I'm sorry, I'll pull this into the `pulibrary` fork and see about preserving our customizations from there.

Comment (tpendragon) [reply public/uv/lib/AVComponent.js]: I could look into getting them upstream too, but it'd probably take a while.

**Comment by jrgriffiniii** [comment]
I've ported local modifications introduced in the following commits:
- https://github.com/pulibrary/figgy/commit/8047613162da0777771df2d4485af2438bc67983#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/5e8bd9293e209d30ddb74be168b3b168ff714670#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/40b4a3a59d30d73ec6afc7d2173f37b890d53e5c#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/ac3c367280b0f2a04b83541c36cd192f16c0825c#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/2729f0148d9d6ecb9a51cf849f07b9bcb44c9390#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/dff4d5bb9acba1f5b29e621e3c216e301c45c957#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/f8246be8cd6912ca9a87ce23d0c0480d75c8aaf6#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/bf185dfe1c263d4089f16daf1921721306f06788#diff-4c9184f37cff01bcdc32dc486ec36961
- https://github.com/pulibrary/figgy/commit/2ce3523df4642a019cf6f1847e2265636fe0fe63#diff-4c9184f37cff01bcdc32dc486ec36961

**Comment by tpendragon** [comment]
I think this is good now, but I'll have to test this on staging and find the audio files that were a problem to make sure it's not skipping.

**Comment by tpendragon** [comment]
@jrgriffiniii I don't know what's wrong, but we can't use this:

https://figgy-staging.princeton.edu/catalog/22425a48-0a5a-4bd9-a9e8-3eb948595ad6 Listen to the end of the fourth track. It should end on a hard syllable “basement” then continue to the next. Instead it’s ending and some other random track is playing between that and the next track, then the next track continues.

**Comment by jrgriffiniii** [comment]
I'm sorry, I'll look to review the customizations I ported.  It may well be the case that I should just close this PR if significant work is needed to get this functioning.

**Comment by tpendragon** [comment]
44 seconds into that fourth track is also starts to hiccup, like it's playing the same spot over and over again.

**Comment by tpendragon** [comment]
Oh geez I think it's doing this in production too, so maybe it's fine.

**Comment by tpendragon** [comment]
@jrgriffiniii This is working better for me than prod, so I'm :+1:

**Pull request merged**